bitnami/cassandra/values.yaml

```diff
@@ -214,6 +214,9 @@ replicaCount: 1
 ##
 updateStrategy:
   type: RollingUpdate
+## @param automountServiceAccountToken Mount Service Account token in pod
+##
+automountServiceAccountToken: false
```

Comment:
I think it is missing the removal of `serviceAccount.automountServiceAccountToken`